lisa/tests/base.py

```diff
@@ -869,6 +870,7 @@ class RTATestBundle(FtraceTestBundle):
     """
 
     @requires_events('sched_switch')
+    @may_use_events(*map((lambda e: 'rtapp_' + e), RTA.default_ftrace))
```

Comment:
Looks like no... will remove it.

Reply:
@douglas-raillard-arm  Wait... no... AFAIU I need that decorator somewhere to automatically add the rt-app generated trace events to the `default_ftrace` collector we use, don't I?

Reply:
Ok, so we don't trace any rt-app event by default if not explicitly asked by the user.

Reply:
We only collect what can actually be used by some methods of the `TestBundle`, so that all the methods of the object will work as expected (provided the events do exist on the kernel, otherwise there is just a warning in the log and it proceeds). Everything else is up to the user to configure (by passing a custom `FtraceCollector` to `from_target()`) if they need it. When using exekall, some user-defined events can be given and they will be merged with the ones that are actually needed by the class.

Reply:
Fixed by using `rtapp_*` events reported by the ftrace_collector in use.